src/@next/Combobox/comboboxStoryHelper/ComboboxMultuSelect.tsx

```diff
@@ -87,7 +87,7 @@ export const ComboboxMultiSelect = ({
   }, [isSearchEmpty, options]);
 
   return (
-    <div style={{ maxWidth: '500px' }}>
+    <div style={{ maxWidth: 500 }}>
```

Comment:
i need to use non-unit number else it will have problem rendering this storybook